with-snowflake/README.md

```diff
@@ -0,0 +1,315 @@
+
+```graphql
+extend type Query {
+  """
+  Looks up employee records by `EMAIL` in the Snowflake table `EMP_BASIC`.
+
+  The specific warehouse, database and schema for `EMP_BASIC` is defined
+  by a DSN in `config.yaml` in the `snowy` configuration.
+  """
+  employee(EMAIL: String!): Employee
+    @dbquery(type: "snowflake", table: "EMP_BASIC", configuration: "snowy")
+}
+```
+
+`Query.employees` uses an identical `@dbquery` but since it follows standard
+GraphQL pagination and has a `filter` argument StepZen automatically provides
+the rich functionality of pagination and filtering.
+
+```graphql
+extend type Query {
+  """
+  pages through employee records from `EMP_BASIC` with optional filtering.
+  Standard GraphQL pagination is used, thus executing `Query.employees`
+  returns a `EmployeeConnection` that contains standdard paging information
+  and the edges that match the filter with their node values and cursor information.
+
+  The specific warehouse, database and schema for `EMP_BASIC` is defined
+  by a DSN in `config.yaml` in the `snowy` configuration.
+  """
+  employees(
+    first: Int! = 5
+    after: String = ""
+    filter: EmployeeFilter
+  ): EmployeeConnection
+    @dbquery(type: "snowflake", table: "EMP_BASIC", configuration: "snowy")
+}
+```
+
+Now you can start to execute requests to see the power of StepZen and Snowflake.
+
+## GraphQL Requests
+
+You can now issue GraphQL requests against your endpoint using your favourite GraphQL client,
+the endpoint URL was displayed by `stepzen deploy` and will be of the form:
+ * `https://account.stepzen.net/api/with-snowflake/__graphql`
+
+Authentication is required (see https://stepzen.com/docs/connecting-frontends/connecting-to-stepzen )
+
+Here we will use `stepzen request` to make requests. `stepzen request` automatically uses
+the correct authentication header and determines the endpoint from your account
+and the current directory.
+
+Execute this command:
+
+```
+stepzen request 'query{employee(EMAIL:"wsizeyf@sf_tuts.com"){FIRST_NAME LAST_NAME CITY}}' 
+```
+
+This execute this GraphQL query operation to find information about a single employee given an email address:
+```graphql
+query
+{
+  employee(EMAIL:"wsizeyf@sf_tuts.com")
+  {
+    FIRST_NAME
+    LAST_NAME
+    CITY
+  }
+} 
+```
+(with `stepzen request` we have compressed operations to simplify copying the command)
+
+The response should be:
+```json
+{
+  "data": {
+    "employee": {
+      "FIRST_NAME": "Wallis",
+      "LAST_NAME": "Sizey",
+      "CITY": "Taibao"
+    }
+  }
+}
+```
+
+We can page through all employees using `Query.employees` and standard GraphQL pagination.
+With StepZen using standard pagination allows clients to handle paging through results
+consistently regardless of if the backend is a database, REST api or another GraphQL API.
+
+First we show how to page through all employees, starting with the first five:
```

Comment:
First we show -> The following request shows how to page through all employees, starting with the first five: